spec/index/index_spec.rb

```diff
@@ -102,6 +120,11 @@
       subject { Daru::Index.new ('a'..'z').to_a  }
       its(:inspect) { is_expected.to eq "#<Daru::Index(26): {a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t ... z}>" }
     end
+
+    context 'index with name' do
+      subject { Daru::Index.new ['one', 'two', 'three'], name: 'number'  }
+      its(:inspect) { is_expected.to eq "#<Daru::Index: number(3): {one, two, three}>" }
```

Comment:
Hm, notice on dataframe format makes sense... But for one-line index inspect I think my proposal will suite better. Two colons are really unreadable :(